Cargo.toml

```diff
@@ -4,7 +4,6 @@ members = [
   "prql-python",
   "prql-compiler",
   "prql-js",
-  "prql-web",
```

Comment:
RIP, this was a journey. Ashes to ashes...

Reply:
Sorry... it will remain in history and our hearts :D

Reply:
😂